manual/assets/js/src/demos/halftone.ts

```diff
@@ -65,7 +69,7 @@ window.addEventListener("load", () => void load().then((assets) => {
 	settings.rotation.sensitivity = 2.2;
 	settings.rotation.damping = 0.05;
 	settings.translation.damping = 0.1;
-	controls.position.set(1, 10, 1);
+	controls.position.set(-1, 10, -1);
```

Comment:
moved the base position around to make the initial view show the effect off more, the original position's view was too dark to notice the effect in action